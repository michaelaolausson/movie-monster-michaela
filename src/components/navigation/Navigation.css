
.navbar {
    margin-bottom: 3em;
}
ul {
    list-style: none;
}
nav ul li {
    display:inline;
    margin-right: 2em;
}
nav ul li a {
    text-decoration: none;
    color: #e1e1e1;
    font-size: 1em;
    text-transform: uppercase;
    font-weight: bold;
}
nav ul li a:active {
    text-decoration: none;
    color: #e1e1e1;
}

h1.start-heading {
  font-family: 'Creepster', cursive;
  font-size: 4em;
  color: #77D64F;
}