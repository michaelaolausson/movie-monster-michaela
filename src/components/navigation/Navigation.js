import React from "react";
import "./Navigation.css";

const Navigation = () => {
  return (
    <nav>
      <ul>
        <li>
          <a className="home-a" href="/">
            <h1 className="start-heading">
              M<span>ovie Monster</span>
            </h1>
          </a>
        </li>
        <li>
          <a href="/search">
            <i className="fa fa-search mr-2"></i> <span>Search</span>
          </a>
        </li>
        <li>
          <a href="/favourites">
            <i className="fa fa-heart-o mr-2"></i> <span>Favourites</span>
          </a>
        </li>
        <li>
          <a href="/watchlist">
            <i className="fa fa-clock-o mr-2"></i> <span>Watchlist</span>
          </a>
        </li>
        <li>
          <a href="/auth">
            <i className="fa fa-user mr-2"></i> <span>Profile</span>
          </a>
        </li>
      </ul>
    </nav>
  );
};

export default Navigation;
