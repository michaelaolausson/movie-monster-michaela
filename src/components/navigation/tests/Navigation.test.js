import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { shallow } from 'enzyme';
import Navigation from '../Navigation.js'

describe('Navigation', () => {
    const wrapper = shallow(<Navigation />);
    it('matches snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    })
})