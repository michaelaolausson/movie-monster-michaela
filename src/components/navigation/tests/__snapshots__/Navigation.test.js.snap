// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation matches snapshot 1`] = `"<nav><ul><li><a class=\\"home-a\\" to=\\"/\\"><h1 class=\\"start-heading\\">M<span>ovie Monster</span></h1></a></li><li><a to=\\"/search\\"><i class=\\"fa fa-search mr-2\\"></i> <span>Search</span></a></li><li><a to=\\"/favourites\\"><i class=\\"fa fa-heart-o mr-2\\"></i> <span>Favourites</span></a></li><li><a to=\\"/watchlist\\"><i class=\\"fa fa-clock-o mr-2\\"></i> <span>Watchlist</span></a></li><li><a to=\\"/auth\\"><i class=\\"fa fa-user mr-2\\"></i> <span>Profile</span></a></li></ul></nav>"`;
